mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -172,6 +172,28 @@ def __init__(self,
         else:
             self.cls_criterion = cross_entropy
 
+    def _get_class_weight(self, class_weight):
```

Comment:
Could we make this an utility function that could be shared by all losses?

Reply:
Yes, good idea!